doc/flame/collision_detection.md

```diff
@@ -229,6 +229,11 @@ Ray casting and ray tracing are methods for sending out rays from a point in you
 being able to see what these rays collide with and how they reflect after hitting
 something.
 
+For all of the following methods, if there are any hitboxes that you which to ignore you can add the
+`ignoreHitboxes` argument which is a list of the hitboxes that you which to disregard for the call.
+This can be quite useful if you are casting rays from within a hitbox, which could be on your player
+or npc for example.
```

Comment:
```suggestion
For all of the following methods, if there are any hitboxes that you wish to ignore, you can add the
`ignoreHitboxes` argument which is a list of the hitboxes that you wish to disregard for the call.
This can be quite useful for example if you are casting rays from within a hitbox, which could be on your player
or NPC; or if you don't want a ray to bounce off a `ScreenHitbox`.
```